comment/mixins.py

```diff
@@ -1,16 +1,40 @@
 import abc
 
 from django.contrib.auth.mixins import LoginRequiredMixin, AccessMixin
-from django.contrib import messages
 from django.core.exceptions import ImproperlyConfigured
-from django.http import HttpResponseBadRequest, HttpResponseForbidden
+from django.http import HttpResponseBadRequest, HttpResponseForbidden, JsonResponse
+from django.views.generic import FormView
 
 from comment.conf import settings
 from comment.utils import is_comment_admin, is_comment_moderator
 from comment.validators import ValidatorMixin
 from comment.messages import ErrorMessage, FlagError, FollowError
 from comment.service.email import DABEmailService
 from comment.messages import EmailInfo
+from comment.forms import CommentForm
+from comment.context import DABContext
+
+
+class DABResponseData:
+    status = 200
+    data = None
+    error = None
+    anonymous = False
+    msg = None
+
+    def json(self):
+        return {
+            'data': self.data,
+            'error': self.error,
+            'anonymous': self.anonymous,
+            'msg': self.msg
+        }
```

Comment:
it could be more intuitive if we could rename this function to something along the lines of `response_json`, so that we get more clear information when we call it from a view.

Reply:
I am more convenient with `json` since the same is used in the default django response `response.json()`.